test/media/streaming_engine_integration.js

```diff
@@ -401,10 +397,6 @@ describe('StreamingEngine', () => {
       streamingEngine.switchVariant(variant);
       await streamingEngine.start();
 
-      // IE is sensitive and throws InvalidStateError when you seek while
-      // readyState is 0.
-      await waiter.timeoutAfter(5).waitForEvent(video, 'loadeddata');
```

Comment:
Without IE, we no longer have to wait for 5 seconds, and we still need to wait for the 'loadeddata' event. 
Let's have `await waiter.waitForEvent(video, 'loadeddata');` instead. Thank you!

Reply:
It didn't wait for 5 seconds, but rather, would timeout and continue after 5 seconds if the 'loadeddata' event did not arrive.

Reply:
Changed!

Reply:
Synced up with Joey and we should actually keep the timeout.  It's a just-in-case measure to make sure the test can't be stuck at that stage for more than 5s.
Shall we keep it as it was? Sorry about my comment earlier :)

Reply:
Done!